<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetMain</class>
 <widget class="QWidget" name="WidgetMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WidgetMain</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Visores</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="labelVisor1">
         <property name="text">
          <string>labelVisor1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="labelVisor2">
         <property name="text">
          <string>labelVisor2</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnListarCamaras">
       <property name="text">
        <string>Listar Cámaras</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbxCamaras"/>
     </item>
     <item>
      <widget class="QCheckBox" name="chbCamara">
       <property name="text">
        <string>Activar/Desactivar Cámara</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnTakePhoto">
     <property name="text">
      <string>Tomar Foto</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
